plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Cloud Tools for Eclipse App Engine New Project Generation
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.newproject;singleton:=true
+Bundle-Version: 0.1.0.qualifier
+Bundle-Vendor: Google, Inc.
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: org.eclipse.jface.viewers,
```

Comment:
<kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>A</kbd> gives a quick way to access all OSGi & Eclipse-related artifacts: bundles, packages, extension points & extensions.  It's all pulled from what's in the target platform.  There are various views (Window > Show View > Plug-in Development) related to introspecting on the target; most useful there is the _Plug-in Registry_.  And then using the Manifest editor and its tabs.  If you want to go deeper, use _File > Import > Plug-in Development > Plug-ins and Fragments_ to import bundles into your environment so they show up in your project list.